clients/client-python/gravitino/utils/http_client.py

```diff
@@ -117,11 +115,10 @@ def _make_request(self, opener, request, timeout=None):
             return opener.open(request, timeout=timeout)
         except HTTPError as err:
             exc = handle_error(err)
-            exc.__cause__ = None
-            raise exc
+            raise exc from None
 
     def _request(
-        self, method, endpoint, params=None, json=None, headers=None, timeout=None
+        self, method, endpoint=None, params=None, json=None, headers=None, timeout=None
```

Comment:
This change is because I found there's a usage `_request("close")` and pass no endpoint as argument, and it make pylint alert.
Maybe modify the usage into `_request("close", None)`?

Reply:
I'm not sure how "close"  works here...but the endpoint argument should be passed a value for constructing a URL, so IMO, a `None` value is a invalid value.

Reply:
Here is the close function:

https://github.com/datastrato/gravitino/blob/b841e58ad3520f4750aa2c6fe52ae97d98a7ccbd/clients/client-python/gravitino/utils/http_client.py#L161-L162

Reply:
I've replaced `self._request("close")` with `self._request("close", "/")` , meaning the endpoint is the host url itself.